src/SampleMods/OWML.EnableDebugMode/EnableDebugMode.cs

```diff
@@ -43,6 +52,14 @@ private void OnEvent(MonoBehaviour behaviour, Events ev)
 				ModHelper.Console.WriteLine("Player spawner loaded!");
 				_playerSpawner = playerSpawner;
 				_isStarted = true;
+				if (Keyboard.current == null)
```

Comment:
would this ever happen?
if we can expect it's never null it would simplify a lot of code

Reply:
Technically it's possible if i.e. keyboard gets disconnected for a moment due to bad contact or something like that. Wouldn't want game to crash in that case

Reply:
it is shrunk now